app/assets/javascripts/active_admin.js.coffee

```diff
@@ -0,0 +1 @@
+#= require active_admin/base
```

Comment:
I think you can delete this file, right @gonzaM ?

Reply:
Yes